src/components/FadableContent/styles.module.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../scss/variables';
+
+.shell {
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(rgba(255, 255, 255, 0), $white);
```

Comment:
I think using `transparent` would be a little more explicit than `rgba(...)`.

Reply:
Safari handles "transparent" in a wacko way. See the first answer here: https://stackoverflow.com/questions/38391457/linear-gradient-to-transparent-bug-in-latest-safari . That's really the only reason I avoided it. I guess Safari doesn't matter? I always like to aim for cross-compat when possible.

Reply:
This is what using `transparent` looks like in today's Safari on macOS Mojave

<img width="929" alt="Screenshot 2019-07-19 10 12 32" src="https://user-images.githubusercontent.com/55398/61545759-ce109200-aa0d-11e9-9fb3-adb5ac10a7c3.png">

Reply:
I can add a comment to the css about it

Reply:
oh mmh, ok then let's keep `rgba` and document it.